src/screens/Auth/RegisterStack/CreatePasswordScreen/CreatePasswordScreen.tsx

```diff
@@ -21,11 +21,13 @@ export default function CreatePasswordScreen(props: Props) {
   const [confirmPassword, setConfirmPassword] = useState('')
   const {navigate} = props.navigation 
   const user = {
-    userTitle: props.navigation .state.params.userTitle,
     companyCode: props.navigation .state.params.companyCode,
     name: props.navigation .state.params.name,
-    email: props.navigation .state.params.email
+    email: props.navigation .state.params.email,
+    userRecord: props.navigation .state.params.userRecord,
   }
+  const [message, setMessage] = useState('Keep this secure! Use 8+ characters')
+  const [messageStyle, setMessageStyle] = useState(styles.regularText)
```

Comment:
these don't seem to get updated ever, so you can just write out this text directly in the <text> component. no need to make these state variables.